app/screens/settings/notification_settings_mobile/notification_settings_mobile_base.js

```diff
@@ -87,19 +87,23 @@ export default class NotificationSettingsMobileBase extends PureComponent {
     };
 
     saveUserNotifyProps = () => {
-        const props = {...this.state};
-
-        Reflect.deleteProperty(props, 'defaultSound');
-        Reflect.deleteProperty(props, 'selectedUri');
-        Reflect.deleteProperty(props, 'shouldBlink');
-        Reflect.deleteProperty(props, 'shouldVibrate');
-        Reflect.deleteProperty(props, 'showMobilePushModal');
-        Reflect.deleteProperty(props, 'showMobilePushStatusModal');
-        Reflect.deleteProperty(props, 'showMobileSoundsModal');
-        Reflect.deleteProperty(props, 'sound');
+        const props = () => {
```

Comment:
This is more complicated than it needs to be. This could just be 
```javascript
    const {
        channel,
        comments,
        desktop,
        desktop_duration,
        email,
        first_name,
        mention_keys,
        push,
        push_status
    } = this.state;

    return {channel, comments, dekstop, desktop_duration, email, first_name, mention_keys, push, push_status, user_id: this.props.currentUser.id};
```

The anonymous function and the Object.assign don't do anything here. I'd also skip the destructuring itself, but that's entirely personal preference